.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "Appium/tests/helpers/mocks"]
 	path = Appium/tests/helpers/mocks
 	url = https://github.com/jasonkarns/bats-mock
+[submodule "Appium/tests/helpers/bats"]
```

Comment:
I added installation via npm but after thinking a bit, I actually believe that in this case it is a less robust solution since tests are now dependent on availability of packages via npm or even simply on having internet connection to install packages, which is not the best solution I think.